index.html

```diff
@@ -1,13 +1,81 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-  <link rel="stylesheet" href="styles/index.css">
-</head>
-<body>
-  <h1>Quiz-App</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz-App start page</title>
+    <link rel="stylesheet" href="styles/index.css" />
+  </head>
+  <body class="app-grid">
+    <header id="#starthome" class="top">
+      <h1 class="top quiz-app-heading">Quiz-App</h1>
+    </header>
```

Comment:
do you want to use BEM in your project? 
If yes I would recommend following:
`<h1 class="top__title">` so it sticks to the `class="top"` of your `header`
and you have used top in the header and in the h1...or is it spezific styling already?

Reply:
hi, yes i want to use bem, but this app-grid storybook structure we built today together to help us show the structure of the page confused a little bit my thoughts for class-names in here, for this reason i named the class "top" more as a reminder